File: SCALA/Cap03/c03ex05.scala
// c03ex05.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var nome: String = ""

    print("Entre seu nome: ")
    nome = readLine()

    println(s"Ola, $nome! Bem-vindo ao estudo de Scala.")

    print("\nAperte <Enter> para encerrar... ")
    readLine()
  }
}

File: DART/Cap03/c03ex05.dart
// c03ex05.dart

import 'dart:io';

void main() {
  String name;

  stdout.write("Entre seu nome: ");
  name = stdin.readLineSync() ?? "";

  print("Ola, $name! Bem-vindo ao estudo de Dart.");

  stdout.write("\nAperte <Enter> para encerrar... ");
  stdin.readLineSync();
}
